static/app/components/group/issueReplayCount.tsx

```diff
@@ -5,21 +5,24 @@ import {Tooltip} from 'sentry/components/tooltip';
 import {IconPlay} from 'sentry/icons';
 import {t, tn} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
+import type {Group} from 'sentry/types';
 import useReplayCountForIssues from 'sentry/utils/replayCount/useReplayCountForIssues';
 import useOrganization from 'sentry/utils/useOrganization';
 import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 
-type Props = {
-  groupId: string;
+type NewType = {
```

Comment:
why is this `NewType` necessary? can you just inline it in props?

Reply:
oops, that's my editor doing some shit.